tests/fault_tolerance/cancellation/test_vllm.py

```
@@ -456,12 +495,16 @@ def test_request_cancellation_vllm_decode_cancel(
                     process=decode_worker,
                     pattern=f"Aborted Request ID: {request_id}",
                     log_offset=decode_log_offset,
+                    max_wait_ms=5000,
+                    poll_interval_ms=50,
                 )
 
                 # Verify frontend log has kill message
                 _, frontend_log_offset = poll_for_pattern(
```

Ruff (0.16.2)

[warning] Unpacked variable `frontend_log_offset` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)